src/routes/conversation/[id]/+server.ts

```diff
@@ -409,6 +409,8 @@ export async function POST({ request, locals, params, getClientAddress }) {
 
 			let hasError = false;
 
+			let buffer = "";
```

Comment:
Should there be the snippet provided below at the end of the entire process just to be sure that all the tokens were seant to the frontend ?

```ts
if(buffer){
	update({
		type: "stream",
		token: buffer,
	});
}
```

Reply:
We send the final message anyway to the front-end so it shouldn't change much but will add it for sure just in case

Reply:
basically, is there a chance `buffer` was non-empty and you receive the end token. so this non-empty `buffer` doesn't end up getting sent to the frontend ?

Reply:
Solved in [6eb3486](https://github.com/huggingface/chat-ui/pull/964/commits/6eb34862d45b43134a2f8a4e906be1e2091e17b0)!